templates/README.md

```diff
@@ -167,10 +167,17 @@ To disable tracking, simply remove ``require('metrics-tracker-client').track();`
 <!--Include any relevant links-->
 
 # Links
-* [Demo on Youtube](https://www.youtube.com/watch?v=Jxi7U7VOMYg)
-* [Watson Node.js SDK](https://github.com/watson-developer-cloud/node-sdk)
-* [Relevancy Training Demo Video](https://www.youtube.com/watch?v=8BiuQKPQZJk)
-* [Relevancy Training Demo Notebook](https://github.com/akmnua/relevancy_passage_bww)
+* [More Shopping List Sample Apps](https://github.com/ibm-watson-data-lab/shopping-list)
+<!--Keep the link to the main repo (above) in your list. Update the remainder of the list based on what's relevant for your specific project.-->
```

Comment:
I would remove this comment. It's not needed anymore and looks like it may be causing some rendering issues.

Reply:
Never mind. This is the template I'm looking at. Please _do_ keep this comment.